orchagent/orch.cpp

```diff
@@ -23,6 +23,15 @@ extern ofstream gRecordOfs;
 extern bool gLogRotate;
 extern string gRecordFile;
 
+void notifyAbort(){
```

Comment:
API name is not aligned to the functionality. This is doing both notify and abort. API gives the indication that it is only notification. Please move abort to original code and use it only to notify or change the name to say `notifyAndAbort`

Reply:
Handled